README.md

```diff
@@ -2,391 +1,312 @@
-                    +     "\"Ireland\","
-                    +       "\"England\"],"
-                    +   "\"validation\":"
-                    +   "{\"valid_responses\":"
-                    +       "[\"England\","
-                    +             "\"Ireland\","
-                    +       "\"France\","
-                    +       "\"Australia\"]}}]}";
-            init = new Init("questions", secString, consumerSecret, new JSONObject(reqString));
-            String questionJson = init.generate();
-            System.out.println(questionJson);
-
-            // Now you can pass questionJson to the initialisation of Questions API
-            // For instance on jsp you might have
-            // <script src="//questions.learnosity.com"></script>
-            // var questionsApp = LearnosityApp.init(questionJson);
-        }catch (Exception e) {
-            System.out.println(e.getMessage());
-        }
-    }
-}
-```
+### **Development install from a Git clone**
+To install from the terminal, run this command:
 
-#### Arguments
-**service**<br>
-A string representing the Learnosity service (API) you want to integrate with. Valid options are:
+    git clone git@github.com:Learnosity/learnosity-sdk-Java.git
 
-* assess
-* author
-* data
-* items
-* questions
-* reports
+### **Alternative method 1: download the zip file**
+Download the latest version of the SDK as a self-contained ZIP file from the [GitHub Releases](https://github.com/Learnosity/learnosity-sdk-Java/releases) page. The distribution ZIP file contains all the necessary dependencies.
 
-**security**<br>
-An Object that includes your *consumer_key* but does not include your *secret*. The SDK sets defaults for you, but valid options are:
+[(Back to top)](#table-of-contents)
 
-* consumer_key
-* domain (optional)
-* timestamp (optional)
-* user_id (optional)
+## Quick start guide
+Let's take a look at a simple example of the SDK in action. In this example, we'll load an assessment into the browser.
 
-^Note – the SDK accepts all Objects which can be parsed into a org.json.JSONObject (e.g. a map, a JSONString, JSONOBject or a Java bean).
+### **Build the .WAR file**
+To start, navigate to the root folder of the project on the command line, and run the following command:
 
-**secret**<br>
-Your secret key, as provided by Learnosity.
+``` bash
+    make quickstart-assessment
+```
 
-**request**<br>
-An optional associative Object of data relevant to the API being used. This will be any data minus the security details that you would normally use to initialise an API.
+This will compile and test the project, building the .WAR file. 
 
-^Note – the SDK accepts all Objects which can be parsed into a org.json.JSONObject (e.g. a map, a JSONString, JSONOBject or a Java bean).
+You will get the following advice on the command line, to either copy the .WAR file into your J2EE servlet container, or run the Docker below to boot the Jetty web server: 
```

Comment:
Codacy has a fix for the issue: [Expected: 0 or 2; Actual: 1](https://app.codacy.com/gh/Learnosity/learnosity-sdk-java/pullRequest?prid=8556170)
```suggestion
You will get the following advice on the command line, to either copy the .WAR file into your J2EE servlet container, or run the Docker below to boot the Jetty web server:
```